src/frontend/shared-components/src/__snapshots__/storyshots.test.js.snap

```diff
@@ -1,5 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Storyshots Button Default 1`] = `
+<div>
+  <button
+    className="normal-blue btn"
+    onClick={[Function]}
```

Comment:
Is this just making sure that there is a function for onclick, not caring about what it is?

Reply:
yup! just a mock function